ui/src/components/lib/file-input/file-input.tsx

```diff
@@ -0,0 +1,77 @@
+import { useRef } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Icon24Upload } from '@vkontakte/icons'
+import { DropZone, Placeholder, VisuallyHidden } from '@vkontakte/vkui'
+
+import styles from './file-input.module.css'
+
+import type { ChangeEvent, DragEvent } from 'react'
+
+type FileInputProps = {
+  accept?: string[]
+  onError?: (message: string) => void
+  onChange?: (file: File) => void
+}
+
+export const FileInput = ({ onChange, onError, accept }: FileInputProps) => {
+  const { t } = useTranslation()
+  const inputRef = useRef<HTMLInputElement>(null)
+
+  const onDragOver = (event: DragEvent) => {
+    event.preventDefault()
+  }
+
+  const onChangeWithValidation = (files: FileList) => {
+    onError?.('')
```

Comment:
maybe 'Validation Error' ?

Reply:
This is just resetting the previous error text before starting the validation